window/src/os/windows/window.rs

```diff
@@ -887,7 +887,7 @@ impl WindowOps for Window {
     fn config_did_change(&self, config: &ConfigHandle) {
         let config = config.clone();
         Connection::with_window_inner(self.0, move |inner| {
-            inner.config_did_change(&config);
+            inner.config_did_change(config);
```

Comment:
Seems like this shouldn't be here either?

```suggestion
            inner.config_did_change(&config);
```